packages/aws-cdk-lib/aws-sns/README.md

```diff
@@ -21,6 +21,26 @@ const topic = new sns.Topic(this, 'Topic', {
 
 Note that FIFO topics require a topic name to be provided. The required `.fifo` suffix will be automatically generated and added to the topic name if it is not explicitly provided.
 
+## Import
```

Comment:
Please don't use the term "import". This is confusing because it doesn't actually import resources into the stack (which people might otherwise think it does).

We prefer the term "referencing existing resources".